core/server/models/base/when.js

```diff
@@ -0,0 +1,59 @@
+var config = require('../../config'),
```

Comment:
I think that this change could be easily moved out into its own PR on top of master, merging with the events file? There's no reason why it can't be merged before this pr?